autogen/main/cluster.tf.tmpl

```diff
@@ -537,6 +537,13 @@ resource "google_container_cluster" "primary" {
         }
       }
 
+      dynamic "fast_socket" {
+        for_each = lookup(var.node_pools[0], "enable_fast_socket", false) ? [true] : []
```

Comment:
I recommend defaulting to `null`, that way the content value can be populated for both `true` and `false`.
```suggestion
        for_each = lookup(var.node_pools[0], "enable_fast_socket", null) != null ? [var.node_pools[0].enable_fast_socket] : []
```

Reply:
fixed